src/content/1.7/development/webservice/reference.md

```diff
@@ -0,0 +1,225 @@
+
+- `search`: only GET and HEAD.
+- `stock_availables`: only GET, POST, and HEAD.
+- `stock_movements`: only GET and HEAD.
+- `stocks`: only GET and HEAD.
+- `supply_order_details`: only GET and HEAD.
+- `supply_order_histories`: only GET and HEAD.
+- `supply_order_receipt_histories`: only GET and HEAD.
+- `supply_order_states`: only GET and HEAD.
+- `supply_orders`: only GET and HEAD.
+- `warehouse_product_locations`: only GET and HEAD.
+- `warehouses`: only GET, POST, PUT, and HEAD.
+
+All resources have two schemas that are accessible via a parameter:
+
+- `?schema=blank`: returns a blank XML tree of the chosen resource.
+- `?schema=synopsis`: returns a blank XML tree of the chosen resource, with the format that is expected for each value and specific indicators (ie, if the node is required, and its maximum size in number of characters).
+
+## Available resources
+
+| Resource | Description |
+|----------|-------------|
+| addresses | The Customer, Manufacturer and Customer addresses |
+| carriers | The Carriers |
+| cart_rules | Cart rules management |
+| carts | Customer's carts |
+| categories | The product categories |
+| combinations | The product combinations |
+| configurations | Shop configuration |
+| contacts | Shop contacts |
+| content_management_system | Content management system |
+| countries | The countries |
+| currencies | The currencies |
+| customer_messages | Customer services messages |
+| customer_threads | Customer services threads |
+| customers | The e-shop's customers |
+| customizations | The product customizations |
+| deliveries | Product deliveries |
+| employees | The Employees |
+| groups | The customer's groups |
+| guests | The guests |
+| image_types | The image types |
+| images | The images |
+| images/general/header | The shop's logo in the header |
+| images/general/mail | The shop's logo in the e-mails |
+| images/general/invoice | The shop's logo in the invoice |
+| images/general/store_icon | The shop's logo as a favicon |
+| images/products | The products images |
+| images/categories | The categories images |
+| images/manufacturers | The manufacturers images |
+| images/suppliers | The suppliers images |
+| images/stores | The stores images |
+| images/customizations | The customizations images |
+| languages | Shop languages |
+| manufacturers | The product manufacturers |
+| messages | The customers messages |
+| order_carriers | The order carriers |
+| order_details | Details of an order |
+| order_histories | The Order histories |
+| order_invoices |The Order invoices |
+| order_payments |The Order payments |
+| order_slip | The Order slips |
+| order_states |The Order states |
+| orders | The Customers orders |
+| price_ranges | Price range |
+| product_customization_fields | The Product customization fields |
+| product_feature_values | The product feature values |
+| product_features | The product features |
+| product_option_values | The product options value |
```

Comment:
Consider being a _little_ more descriptive 😛

Reply:
It was originally copied from the initial doc 😅 I tried to explain a bit more when I thought it was useful But sometimes I don't know what to say more (like Products are Products ^^)
Let me know if you think some need extra details